front-end/src/host/HostApp.tsx

```diff
@@ -202,44 +209,47 @@ export default function HostApp(props: IHostProps) {
 
   return (
     <div className="scroll">
-      <PlayAudio src={theme} loop={true} />
-      <div className="banner">
-        <div className="musicButton">
-          <IconButton onClick={() => muteMusic(muted)}>
-            <img className="musicIcon" src={muted ? musicOff : musicOn} />
-          </IconButton>
-        </div>
-        <div className="hostFormat">
-          <img className="logohost" src={logo} />
+      <AddAnnouncementContext.Provider value={addAnnouncement}>
```

Comment:
This wrapper provides the context for `addAnnouncement` to every descendent component.